core/src/com/unciv/models/ruleset/Policy.kt

```diff
@@ -33,6 +35,19 @@ open class Policy : RulesetObject() {
         fun isBranchCompleteByName(name: String) = name.endsWith(branchCompleteSuffix)
     }
 
+    fun matchesFilter(filter: String): Boolean {
+        return MultiFilter.multiFilter(filter, ::matchesSinlgeFilter)
+    }
+    fun matchesSinlgeFilter(filter: String): Boolean {
+        return when(filter) {
+            in Constants.all -> true
+            name -> true
+            branch.name + "branch" -> true
```

Comment:
This won't catch

Reply:
Is it because I forgot the space?

Reply:
Also the filter contains square brackets, no?

Reply:
No? Actually... Huh... It probably should